support-modules/aws/src/main/scala/com/gu/aws/AwsCloudWatchMetricPut.scala

```diff
@@ -23,8 +23,17 @@ object AwsCloudWatchMetricPut {
     namespace: MetricNamespace,
     name: MetricName,
     dimensions: Map[MetricDimensionName, MetricDimensionValue],
-    value: Double = 1.0
+    value: Double
   )
+  object MetricRequest {
+
+    def apply(
+      namespace: MetricNamespace,
+      name: MetricName,
+      dimensions: Map[MetricDimensionName, MetricDimensionValue]
+    ): MetricRequest = new MetricRequest(namespace, name, dimensions, 1.0)
```

Comment:
tasty-reader fell over on default args in case classes but I've no idea why.

Reply:
that's not great!

Reply:
for the record, the error was, 
```
[error] /Users/john_duffell/code/support-frontend/support-services/src/main/scala/com/gu/aws/AwsCloudWatchMetricSetup.scala:57:18: not enough arguments for method apply: (namespace: com.gu.aws.AwsCloudWatchMetricPut.MetricNamespace, name: com.gu.aws.AwsCloudWatchMetricPut.MetricName, dimensions: scala.collection.immutable.Map[com.gu.aws.AwsCloudWatchMetricPut.MetricDimensionName,com.gu.aws.AwsCloudWatchMetricPut.MetricDimensionValue], value: Double): com.gu.aws.AwsCloudWatchMetricPut.MetricRequest in object MetricRequest
[error]     MetricRequest(
[error]                  ^
```

Reply:
https://github.com/lampepfl/dotty/issues/12704